server/container/util-java8/src/main/java/org/apache/james/util/mime/MessageContentExtractor.java

```diff
@@ -173,7 +173,9 @@ private boolean isNotAttachment(Entity part) {
     }
 
     private boolean isInlinedWithoutCid(Entity part) {
-        return part.getDispositionType().equals(MimeMessage.INLINE) && part.getHeader().getField(CONTENT_ID) == null;
+        return part.getDispositionType() != null
```

Comment:
Can't we rely on Objects.equals(part.getDispositionType(), MimeMessage.INLINE) ?